crates/volta-core/src/tool/node/resolve.rs

```diff
@@ -253,7 +256,11 @@ fn resolve_node_versions(url: &str) -> Fallible<RawNodeIndex> {
             let expiry = create_staging_file()?;
             let mut expiry_file: &File = expiry.as_file();
 
-            write!(expiry_file, "{}", expires).with_context(|| {
+            let mut header_values = Vec::with_capacity(1);
+            expires.encode(&mut header_values);
+            let encoded_expires = header_values.first().unwrap().to_str().unwrap();
```

Comment:
```suggestion
            // Since we just `.encode()`d into `header_values
            let encoded_expires = header_values.first().unwrap().to_str().unwrap();
```